proposals/background-scanning-v2.md

```diff
@@ -0,0 +1,115 @@
+# Meta
+[meta]: #meta
+- Name: Background Scanning V2
+- Start Date: Jan 07 2025
+- Update data (optional): (fill in today's date: YYYY-MM-DD)
+- Author(s): vishal-chdhry
+- Supersedes: N/A
+
+# Table of Contents
+[table-of-contents]: #table-of-contents
+- [Meta](#meta)
+- [Table of Contents](#table-of-contents)
+- [Overview](#overview)
+- [Definitions](#definitions)
+- [Motivation](#motivation)
+- [Proposal](#proposal)
+- [Implementation](#implementation)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Prior Art](#prior-art)
+- [Unresolved Questions](#unresolved-questions)
+
+# Overview
+
+This KDP proposes an enhancement for Kyverno's background scanning feature.
+
+# Definitions
+
+- **Background Scanning**: Background scanning is a kyverno feature which allows Kyverno to periodically scan existing resources and find if they match any validate or verifyImages rules.
+
+# Motivation
+
+- Why should we do this?
+- What use cases does it support?
+- What is the expected outcome?
+
+# Proposal
+
+## Approach 1: Use Cron to schedule periodic scans on each policy
+
+Kubernetes has pod controller called CronJob which creates Jobs on a repeating schedule. According to Kubernetes documentation:
+
+> CronJob is meant for performing regular scheduled actions such as backups, report generation, and so on.
+
+This approach for scheduling scans can be used by Kyverno policies as follows:
+
+```yaml
+apiVersion: kyverno.io/v3alpha1
```

Comment:
What is the desired version? I see `apiVersion: kyverno.io/v1alpha1` in the original KDP.